src/styles/utils/variables.scss

```diff
@@ -0,0 +1,3 @@
+$main-f-color: #060b35;
+$secondary-f-color: #616070;
+$blue-accent: #00acdc;
```

Comment:
You need to fix the variable names, they should show their purpose, not their content. if I decided to change the color, I would have to change it everywhere, not just in the variable definition.

Also, don't give variables unreadable names.